.gitignore

```diff
@@ -2,3 +2,5 @@
 pyproject.toml
 .DS_Store
 .tool-versions
+
```

Comment:
Nit pick, but we don't need newline here

Reply:
I need a newline here